runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerOverlay.java

```diff
@@ -62,6 +62,9 @@
 		ItemID.RED_SLAYER_HELMET_I,
 		ItemID.TURQUOISE_SLAYER_HELMET,
 		ItemID.TURQUOISE_SLAYER_HELMET_I,
+		ItemID.TWISTED_SLAYER_HELMET,
+		ItemID.TWISTED_SLAYER_HELMET_I,
+		ItemID.TWISTED_HORNS,
```

Comment:
The horns should not be here, since they do not have the functionality of the Slayer Gem